pkg/chaosctl/common/exec.go

```diff
@@ -75,29 +75,30 @@ func Exec(ctx context.Context, pod v1.Pod, cmd string, c *kubernetes.Clientset)
 }
 
 // ExecBypass use chaos-daemon to enter namespace and execute command in target pod
-func ExecBypass(ctx context.Context, pod v1.Pod, daemon v1.Pod, cmd string, c *kubernetes.Clientset) (string, error) {
-	cmdSubSlice := strings.Fields(cmd)
-	if len(cmdSubSlice) == 0 {
-		return "", fmt.Errorf("command should not be empty")
-	}
+func ExecBypass(ctx context.Context, pod v1.Pod, daemon v1.Pod, cmd string, c *kubernetes.Clientset, nsTypes ...bpm.NsType) (string, error) {
```

Comment:
Since we already use the bpm(which works with "nsexec"), I think it's OK to enter all common namespaces(mnt, pid, network, etc.) in `ExecByPass`. And then we could delete the last parameter ` nsTypes ...bpm.NsType`.

Reply:
Yeah I could not find side effects when we enter all possible namespaces. Fixed thx!